e2e/doc/code_layout.txt

```diff
@@ -0,0 +1,44 @@
+e2e
+├── doc: Various documents
+│   ├── code_layout.txt
+│   └── condition_table.md
+├── keywords: Keyword definitions for workloads
+│   │   ├── __init__.py
+│   │   ├── common_keywords.py
```

Comment:
The naming is wrong. Should be *.resource

Reply:
I've re-update the code structure within `tree` command.